packages/core/src/components/Link/Link.tsx

```diff
@@ -25,6 +25,12 @@ type Props = ComponentProps<typeof MaterialLink> &
  * Thin wrapper on top of material-ui's Link component
  * Makes the Link to utilise react-router
  */
-export const Link = React.forwardRef<any, Props>((props, ref) => (
-  <MaterialLink ref={ref} component={RouterLink} {...props} />
-));
+export const Link = React.forwardRef<any, Props>((props, ref) => {
+  const to = props.to.toString();
```

Comment:
`String(props.to)` is likely a bit safer here. Less crashy and more just navigate to`/undefined` xP

Reply:
Still enjoying the new world of Typescript 😄  Thanks for the hint!